simulator-ui/src/main/resources/static/package.json

```diff
@@ -15,10 +15,12 @@
   },
   "private": true,
   "dependencies": {
+    "@angular/animations": "^13.0.2",
     "@angular/common": "^13.0.2",
     "@angular/compiler": "^13.0.2",
     "@angular/core": "^13.0.2",
     "@angular/forms": "^13.0.2",
+    "@angular/material": "^13.0.2",
```

Comment:
it's interesting to have bootstrap and material on the classpath 😅

Reply:
Why?

Reply:
well, two different css frameworks..